crypto/x509/x_all.c

```diff
@@ -41,18 +41,26 @@ int NETSCAPE_SPKI_verify(NETSCAPE_SPKI *a, EVP_PKEY *r)
 
 int X509_sign(X509 *x, EVP_PKEY *pkey, const EVP_MD *md)
 {
-    x->cert_info.enc.modified = 1;
-    return (ASN1_item_sign(ASN1_ITEM_rptr(X509_CINF), &x->cert_info.signature,
+    int ret = 0;
+
+    ret = ASN1_item_sign(ASN1_ITEM_rptr(X509_CINF), &x->cert_info.signature,
                            &x->sig_alg, &x->signature, &x->cert_info, pkey,
-                           md));
+                           md);
```

Comment:
Don't forget to re-align the indentation (in doubt, see the coding guidelines).
I recommend using
```
util/check-format.pl crypto/x509/{x_all,x509_req}.c
```
where you may of course ignore the findings on lines that you did not touch.